packages/seda-chain-contracts/src/data_request.rs

```diff
@@ -33,26 +28,25 @@ pub mod data_requests {
     }
 
     /// Posts a data request to the pool
+    #[allow(clippy::too_many_arguments)]
```

Comment:
I also put this comment in pr #45 but we should pass the PostDataRequest struct instead of having a lot of args